template/template_func.go

```diff
@@ -68,3 +69,18 @@ func areaPath(parentPath, name string) string {
 	}
 	return fmt.Sprintf("%v/%v", lower(p), lower(name))
 }
+
+func jsonToArray(value interface{}) []interface{} {
+	m := []interface{}{}
+	if jsondata, ok := value.(string); ok {
+		err := json.Unmarshal([]byte(jsondata), &m)
+		if err != nil {
+			return make([]interface{}, 0, 0)
+		}
+	}
+	return m
+}
```

Comment:
new function added for template, used at [link](https://github.com/fabric8-services/fabric8-notification/pull/75/files#diff-6ef8b43b827fe67fe3f4b38360299a68R88).  This will convert `vulnerable_deps` to array.